pkg/ansible/ansible.go

```diff
@@ -30,11 +31,17 @@ func Playbook(dir, inventory, extraVars, playbook string) (int, error) {
 	c.Stdout = os.Stdout
 	c.Stderr = os.Stderr
 	if err := c.Run(); err != nil {
+		// Try to get the exit code
 		if exitError, ok := err.(*goexec.ExitError); ok {
 			return exitError.ExitCode(), err
+		} else {
+			// This will happen if ansible is not available in $PATH
+			return defaultFailedCode, err
 		}
+	} else {
+		// successful execution of ansible playbook
+		return 0, nil
 	}
-	return 0, nil
```

Comment:
Not sure golang will work fine without any return statement in the end

Reply:
It did work. May be because all the internal conditions have a return.